benchmarks/src/lib.rs

```diff
@@ -21,5 +21,4 @@ pub mod imdb;
 pub mod parquet_filter;
 pub mod sort;
 pub mod tpch;
-mod util;
-pub use util::*;
+pub mod util;
```

Comment:
this seems like an improvment to me (not re-export everything from `util` into the `banchmark` crate namespace